package.json

```diff
@@ -71,5 +71,14 @@
     "Alejandro (https://github.com/Minjung)",
     "Eli Skeggs <skeggse@gmail.com> (https://github.com/skeggse)",
     "Bent Cardan <bent@nothingsatisfies.com> (https://github.com/reqshark)"
+  ],
+  "files": [
+    "HISTORY.md",
+    "LICENSE",
+    "README.md",
+    "index.js",
+    "package.js",
+    "lib/",
+    "build/Release/zmq.node"
```

Comment:
This should not include `build/Release/zmq.node` - this file would only be the one system we publish from (which would mean that, for us, you're going to end up with an OS X node file when you need one for Windows).